### DISCUSSION TITLE
Discord Server!

### DISCUSSION THREAD

**Discussion by Jonneal3** [mckaywrigley/chatbot-ui#1540] [Ideas]
Hi guys - Just created a discord server where we can chat about this stuff more. 

Go here - https://discord.gg/bQeGME3U

@mckaywrigley let me know anytime if you want admin access or whatnot - just need a more "chat-ish" type place to talk about updates and things like that

**Comment by krishagel** [comment]
This invite doesn't appear to be good any longer, do you have a new one? Is the discord still active?

**Reply by Jonneal3** [reply]
Woops may have pseted the wrong one - try this -- https://discord.gg/bQeGME3U @krishagel

**Comment by haydenkong** [comment]
@krishagel the link appears to be expired?????????????

**Comment by Jonneal3** [comment]
@PixelVerseIT Weird. They must have expiring invites for some reason -- try this -- https://discord.com/channels/1202640116955357205/1202640116955357208

**Comment by haydenkong** [comment]
That's not an invite. Discord has a default 7 day expiry for invite links. Click edit link to make the invite link never expire: 
![image](https://github.com/mckaywrigley/chatbot-ui/assets/115069692/469849f3-3530-4608-8442-56b93effffbd)

**Comment by secondtruth** [comment]
@PixelVerseIT @Jonneal3 Here is an alternative invite link, set to not expire: https://discord.gg/bm89NuxyHX
